spec.html

```diff
@@ -108,11 +108,34 @@ <h1>
 			<dd>a wrapped function exotic object _F_</dd>
 		</dl>
 		<emu-alg>
+			1. Let _callerContext_ be the running execution context.
 			1. Let _target_ be _F_.[[WrappedTargetFunction]].
 			1. Assert: IsCallable(_target_) is *true*.
-			1. Let _targetRealm_ be ? GetFunctionRealm(_target_).
-			1. Let _callerRealm_ be ? GetFunctionRealm(_F_).
+			1. Let _calleeContext_ be PrepareForWrappedFunctionCall(_F_).
```

Comment:
well, initially I had something similar to this, and after some back and forward with @syg, we decided that this was not necessary. I will try to dig that discussing out from some of the earlier PRs.